themes/default/content/_index.md

```diff
@@ -84,9 +238,9 @@ deploy:
       description: Scale up your delivery with [advanced automation](/automation/) as you grow.
 
 manage:
-  title: Manage
+  title: Get to market first
```

Comment:
Instead of "Get to market", I wonder if we should emphasize the way a practitioner would think about things ...